examples/nextjs/src/components/shared/contentlets.js

```diff
@@ -1,54 +1,60 @@
-"use client";
-import React from "react";
-import Image from "next/image";
-import Contentlet from "./contentlet";
+'use client';
+import React from 'react';
+import Image from 'next/image';
+import { editContentlet } from '@dotcms/client';
 
 const dateFormatOptions = {
-    year: "numeric",
-    month: "long",
-    day: "numeric",
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric'
 };
 
 function Contentlets({ contentlets }) {
     return (
         <ul className="flex flex-col gap-7">
             {contentlets.map((contentlet) => (
-                <Contentlet contentlet={contentlet} key={contentlet.identifier}>
-                    <li className="flex gap-7 min-h-16">
+                <li className="flex gap-7 min-h-16 relative" key={contentlet.identifier}>
+                    <button
```

Comment:
Show only in `edit mode`

Reply:
That logic will be handled in the component when I create it